specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/Metadata.json

```diff
@@ -0,0 +1,777 @@
+              {
+                "value": "developer"
+              },
+              {
+                "value": "community"
+              }
+            ]
+          }
+        },
+        "name": {
+          "description": "Name of the support contact. Company or person.",
+          "type": "string"
+        },
+        "email": {
+          "description": "Email of support contact",
+          "type": "string"
+        },
+        "link": {
+          "description": "Link for support help, like to support page to open a ticket etc.",
+          "type": "string"
+        }
+      }
+    },
+    "metadataDependencies": {
+      "description": "Dependencies for the content item, what other content items it requires to work.  Can describe more complex dependencies using a recursive/nested structure. For a single dependency an id/kind/version can be supplied or operator/criteria for complex dependencies.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Id of the content item we depend on",
+          "$ref": "#/definitions/metadataId"
+        },
+        "kind": {
+          "description": "Type of the content item we depend on",
+          "$ref": "#/definitions/metadataKind"
+        },
+        "version": {
+          "description": "Version of the the content item we depend on.  Can be blank, * or missing to indicate any version fulfills the dependency.  If version does not match our defined numeric format then an exact match is required.",
+          "$ref": "#/definitions/metadataVersion"
+        },
+        "name": {
+          "description": "Name of the content item",
+          "type": "string"
+        },
+        "operator": {
+          "description": "Operator used for list of dependencies in criteria array.",
+          "type": "string",
+          "enum": [
+            "AND",
+            "OR"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "operator",
+            "values": [
+              {
+                "value": "AND"
+              },
+              {
+                "value": "OR"
+              }
+            ]
+          }
+        },
+        "criteria": {
+          "description": "This is the list of dependencies we must fulfill, according to the AND/OR operator",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/metadataDependencies",
+            "description": "Dependencies for the content item, what other content items it requires to work.  Can describe more complex dependencies using a recursive/nested structure. For a single dependency an id/kind/version can be supplied or operator/criteria for complex formats."
+          }
+        }
+      }
+    },
+    "metadataProperties": {
+      "description": "Metadata property bag.",
+      "required": [
+        "id",
+        "parentId",
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
Top-level properties like 'id', 'kind' should be repeated under properties bag. Please rename the properties. For example: consider renaming 'id' to 'metadataId'.

Reply:
Please refer rpc here: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope

Reply:
Okay I can make this change